File: ProjectFolders/DB_Inserter_Slave/DB_Inserter_Slave/JsonMessage.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DB_Inserter_Slave
{
    public class JsonMessage
    {
        public string username;
        public string post_type;
        public string pwd_hash;
        public string post_title;
        public string post_url;
        public int post_parent;
        public int hanesst_id;
        public string post_text;
    }
}

File: Production/GoInserter/JsonMessage.go
package main

type JsonMessage struct {
	Username string `json:"username"`
	Post_type string `json:"post_type"`
	Pwd_hash string `json:"pwd_hash"`
	Post_title string `json:"post_title"`
	Post_parent int `json:"post_parent"`
	Hanesst_id int `json:"hanesst_id"`
	Post_text string `json:"post_text"`
}